docs/book/src/developer/core/tilt.md

```diff
@@ -85,6 +85,8 @@ NB: the default is dynamic and will be "podman" if the string "Podman Engine" is
 **enable_providers** (Array[]String, default=['docker']): A list of the providers to enable. See [available providers](#available-providers)
 for more details.
 
+**disable_providers** (Array[]String, default=[]): A list of the providers to disable. By default, the `core` provider is always installed. You can also disable it.
+
```

Comment:
Let's instead just add a bool `disable_core_provider`. All others are already opt-in.

Reply:
Yes, sure, it was my original approach, but tried to do more flexible. 
Ok, let's revert it.

Reply:
I also wasn't sure, but I think Christian's suggestion is fine. I don't foresee us adding any other default providers

Is it also possible to have enable_core_provider and default it to true?

Reply:
Yes, fixed.

Added disable_core_provider option which is False by default.